ePay/BANNO.EPAY.V1.POW JSON

```diff
@@ -0,0 +1,77 @@
+#JSON contract for BANNO.EPAY.V1.POW
+
+NOTES: 1. Every state checks memoMode - if memoMode is true, we cannot proceed.
+
+## PRELOADDATA, ## PROCESSDATA
+Because the Specfile functionality is basically the same in both states,
+the same returns will apply to both states. The state name will indicate whether
+the transaction will be performed as a pre-auth or as a final tranaction.
+```json
+{
+  "rgState": "[PRELOADDATA,PROCESSDATA]",
```

Comment:
Can we change the states to `AUTHORIZEPAYMENT` and `PERFORMPAYMENT`?

Reply:
Done